src/main/java/org/schemaspy/output/dot/schemaspy/name/DefaultName.java

```diff
@@ -0,0 +1,18 @@
+package org.schemaspy.output.dot.schemaspy.name;
+
+/**
+ * Decorates a name with the default name.
+ */
+public final class DefaultName implements Name {
```

Comment:
I wonder if it's easy within the domain to understand what a DefaultName is?

Default are generally tricky. Will default trigger if a value has been given?  

```
new Degree(
        twoDegreesOfSeparation,
        new DefaultName(
                new Implied(
                        includeImplied,
                        new EmptyName()
                )
        )
)
```

Reply:
I agree, "default" doesn't convey much information. Still, I found it to be more meaningful than e.g., `RelationshipsDiagram`. All DOT diagram names I've found in this part of the code base had this name (plus som extra bits details), so to me it seemed like some sort of default. If you can provide me with a class name than makes more sense, I'll be happy to change it!

Good point: `DefaultName` should probably not take any parameters. Instead, we could introduce something along `Concatenation` and/or `Priority` in [demonstrated here](https://github.com/jesperolsson-se/FizzBuzz-OCP-Challenge/tree/main/src/main/java/org/example/rearrange/positive).

Reply:
I'm in favor of mergin this and creating an issue for future improvement just so that it's not forgotten.

I don't trust myself in remembering. So I would like to create the issue first.

I would like your input on the following:

We remove the name RelationshipDiagram completely. It's redudent in the fact that we produce ERD.

We change the naming convention or post/pre-fix so that we could start with a BaseName. I'm gonna look into this a bit. It would strange to have default or base in the middle so either be in the beginning or end of the "nestling".

Reply:
> I would like to create the issue first.

Sure, no worries!

> We remove the name RelationshipDiagram completely. It's redudent in the fact that we produce ERD.

I agree, but don't really have a strong opinion.

> We change the naming convention or post/pre-fix so that we could start with a BaseName.

My stance is that pre-fix and post-fix (though I dislike the names) should be classes in their own right, more specifically [Composite](https://en.wikipedia.org/wiki/Composite_pattern)s. The base name and the "qualifiers" would be leafs (i.e., no longer decorators).